extensions/blocks/instagram-gallery/constants.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+export const IS_CURRENT_USER_CONNECTED_TO_WPCOM = get( window.Jetpack_Editor_Initial_State, [
```

Comment:
Not a blocker for now, but maybe we can move this to a shared function as I imagine other blocks may need this in the future?

Reply:
This is just basically getting a global variable, not sure it's worth as a shared utility 😅

Reply:
True, but this may be duplicated in quite a few places in the future. Still, I suppose we can wait until this actually happens :)